modules/apps/data-engine/data-engine-test/src/testFunctional/tests/macros/DataEngineUpgrade.macro

```diff
@@ -76,9 +76,9 @@ definition {
 			locator1 = "DataEngineBuilder#FORM_FIELD_NESTED");
 
 		AssertTextEquals(
-			fieldLabel = "Numeric",
+			fieldLabel = "Number",
```

Comment:
Hey @danvb112 , next time you can amend this kind of fixes on the commit where you wrote the test. Thanks!